logging/src/main/scala/com/ovoenergy/effect/Logging.scala

```diff
@@ -31,17 +31,23 @@ object Logging {
     def tags: Tags
   }
 
-  case class Debug(of: String, tags: Tags = Map.empty) extends Log
-  case class Info(of: String, tags: Tags  = Map.empty) extends Log
-  case class Error(of: String, tags: Tags = Map.empty) extends Log
+  case class Debug(of: String, tags: Tags                 = Map.empty) extends Log
+  case class Info(of: String, tags: Tags                  = Map.empty) extends Log
+  case class Error(of: Ior[String, Throwable], tags: Tags = Map.empty) extends Log
+
+  object Error {
+    def apply(s: String): Error = Error(Ior.Left(s))
```

Comment:
shouldnt the `apply` also take `tags`?

Reply:
Oh yes, you're right I totally missed the tags 👍

Reply:
Addressed in cf092da.